srcpkgs/radicale/INSTALL.msg

```diff
@@ -1,7 +1,13 @@
-You are installing the legacy version of radicale (1.x)!
-Please install the radicale2 package for the latest (2.x) version.
+radicale has been upgraded from 1.x to 3.x, and the radicale2 package
+is now a transitional package that points to the main radicale 3.x
+package. To remove radicale2 and transition to the radicale package:
 
-If you need to migrate data, take the following steps:
+    xbps-pkgdb -m manual radicale
+    xbps-remove radicale2
```

Comment:
I don't think this last part is necessary. Having a dummy package around isn't an issue.

Reply:
Not an issue per se, no. But it makes the system cleaner.

Reply:
Yes, but it also makes an already long INSTALL.msg even longer.

Reply:
This install msg needs to be huge, I don't think we should make it bigger for something that brings little value. People who care about the amount of packages they have know how to fix this.

Reply:
This whole `INSTALL.msg` should be gutted. The last version of radicale v1 is three years old. We should not try to hold users' hands by dumping long instructions that are unlikely to be read anyway. The entire contents of this file should be eliminated and replaced with this simple message:
```
CAUTION: Radicale 3.x is not backwards compatible with Radicale 1.x. If you are upgrading from Radicale 1.x, please consult upstream documentation for information about migrating existing data to the format used by this version.
```
Note that https://radicale.org/1to2 now returns 404, so even Radicale is not interested in helping users migrate from a long-dead version. This is not our problem to solve. If somebody is interested enough, the information can be found in the Wayback Machine.